src/InputWithLabel.js

```diff
@@ -0,0 +1,26 @@
+
+import React from "react";
+   const InputWithLabel = ({
+      value,
+      handleTitleChange,
+      isFocused,
+      children
+   }) =>{
+      
+   const inputRef = React.useRef();
+   
+   React.useEffect(() => {
+      if (isFocused && inputRef.current) {
+      inputRef.current.focus();
+      }
+      }, [isFocused]);
+
+   return(
+    <>
+        <label htmlFor="">{children}</label>
```

Comment:
Is good practice to add id prop in the input element and add that in the htmlFor prop. Something like this:

`return (
    <>
      <label htmlFor="todoTitle">{children}</label>
      <br />
      <input
        name="title"
        id="todoTitle"
        type="text"
        value={value}
        ref={inputRef}
        onChange={handleTitleChange}
      />
    </>
  )`